scripts/setup

```diff
@@ -25,7 +25,13 @@ pipenv run invenio db init create
 pipenv run invenio index destroy --force --yes-i-know
 pipenv run invenio index init --force
 pipenv run invenio index queue init purge
+if [ $COOKIECUTTER_FILE_STORAGE -eq "S3" ]
+then
+docker-compose exec s3 bash -c "mc mb default-bucket" &>/dev/null;
+pipenv run invenio files location s3-default s3://default-bucket --default
+else
 pipenv run invenio files location --default 'default-location'  $(pipenv run invenio shell --no-term-title -c "print(app.instance_path)")
```

Comment:
These have to be changed to 

```console
pipenv run invenio files location create ...
```

since invenio-files-rest 1.1.0 is used.

But this is really because we didn't catch the breaking change nature of invenio-files-rest 1.1.0 ! A potential better solution would be to make a patch release on invenio-files-rest to keep backward compatibility of the cli interface. I made it here: https://github.com/inveniosoftware/invenio-files-rest/pull/242

And this was not caught by the tests because as far as I can tell `scripts/setup` is not called anywhere (anymore) in this repository ¯\_(ツ)_/¯

Reply:
There is no end2end tests yet. That will arrive with matureness of RDM, now things change too fast for that to be useful in a "workload vs usefulness" balance.

let's discuss the invenio-files-rest in a chat.